src/Orleans/Configuration/Options/StatisticsOptions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using System;
+
+namespace Orleans.Configuration
+{
+    public class StatisticsOptions
+    {
+        /// <summary>
+        /// The MetricsTableWriteInterval attribute specifies the frequency of updating the metrics in Azure table.
```

Comment:
I realize these are copied from existing code, but they should probably all be rewritten. For one, none of these are attributes in the C# sense.

Reply:
Good catch thanks, I also remove the Statistics prefix from the prop names.